src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -720,6 +720,120 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     </EmbeddedResource>
   </ItemDefinitionGroup>
 
+  <!--
+    ========================
+    DeterministicSourcePaths
+    ========================
+    
+    Unless specified otherwise enable deterministic source root when building deterministically on CI server, but not for local builds.
+    In order for the debugger to find source files when debugging a locally built binary the PDB must contain original, unmapped local paths.
+  -->
+  <PropertyGroup>
+    <DeterministicSourcePaths Condition="'$(DeterministicSourcePaths)' == '' and '$(Deterministic)' == 'true' and '$(ContinuousIntegrationBuild)' != ''">true</DeterministicSourcePaths>
```

Comment:
`'$(ContinuousIntegrationBuild)' != ''` seems strange.  What if it is set to `false`?

It seems that the condition should be `'$(ContinuousIntegrationBuild)' == 'true'`.

Reply:
Yes! Good catch.